File: android/app/src/main/java/com/coboltforge/dontmind/multivnc/MetaList.java
package com.coboltforge.dontmind.multivnc;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "META_LIST")
public class MetaList{

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "_id")
    public long id;

    @ColumnInfo(name = "NAME")
    public String name;
}

File: android/app/src/main/java/com/coboltforge/dontmind/multivnc/db/MetaList.kt
package com.coboltforge.dontmind.multivnc.db

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.serialization.Serializable

@Serializable
@Entity(tableName = "META_LIST")
data class MetaList(
        @JvmField
        @PrimaryKey(autoGenerate = true)
        @ColumnInfo(name = "_id")
        var id: Long = 0,

        @JvmField
        @ColumnInfo(name = "NAME")
        var name: String? = null
)